api/resolvers/notifications.js

```diff
@@ -228,6 +228,23 @@ export default {
             ORDER BY "sortTime" DESC
             LIMIT ${LIMIT})`
         )
+        queries.push(
+          `(SELECT "Invoice".id::text, "Invoice"."confirmedAt" AS "sortTime", FLOOR("msatsReceived" / 1000) as "earnedSats",
+            'InvoicePaid' AS type
+            FROM "Invoice"
+            WHERE "Invoice"."userId" = $1
+            AND "confirmedAt" IS NOT NULL
+            AND "actionState" = 'PAID'
+            AND "actionType" = 'TRANSFER'
+            AND created_at < $2
+            AND NOT EXISTS (
+              SELECT 1
+              FROM "InvoiceForward"
+              WHERE "InvoiceForward"."invoiceId" = "Invoice".id
+            )              
+            ORDER BY "sortTime" DESC
+            LIMIT ${LIMIT})`
```

Comment:
the current code does not select these actions that are not associated with an item and not p2p, so we add this code here that does just that but only for the actionType=TRANSFER